scalableminds/webknossos
# clearer message when annotation couldnt be found while logged out

### URL of deployed dev instance (used for testing):
- https://annotationnotfoundmessage.webknossos.xyz

### Steps to test:
- create annotation, copy link
- log out, open link, message should suggest logging in
- if the annotation *actually* doesn’t exist, the same message should come up (so that we don’t leak existance metadata)

### Issues:
- fixes #3358

------
